pkg/disk/filesystem.go

```diff
@@ -1,12 +1,57 @@
 package disk
 
 import (
+	"encoding/json"
+	"fmt"
 	"math/rand"
 	"reflect"
 
 	"github.com/google/uuid"
 )
 
+type MkfsOption int
+
+const ( // MkfsOption type enum
+	MkfsVerity MkfsOption = iota // Enable fs-verity option if needed (typically for EXT4)
+)
+
+func getMkfsOptionMapping() []string {
+	return []string{"verity"}
+}
+
+// ToString converts MkfsOption into a human readable string
+func (option MkfsOption) ToString() string {
```

Comment:
I'm a bit late to the party, sorry! I was thinking maybe `String()` is slightly better here, I opened https://github.com/osbuild/images/pull/1545